clouddatabasesv5/cloud_databases_v5.go

```diff
@@ -15,7 +15,7 @@
  */
 
 /*
- * IBM OpenAPI SDK Code Generator Version: 99-SNAPSHOT-f9bb1b47-20210329-151039
+ * IBM OpenAPI SDK Code Generator Version: 99-SNAPSHOT-05af0f12-20220210-122824
```

Comment:
Technically, you should be using an official version of the SDK generator, rather than a locally-built version.